host-src/tool/dc-tool.c

```diff
@@ -284,7 +299,11 @@ char serial_getc()
     retval = serial_read(&tmp, 1);
     if (retval == -1) {
         printf("serial_getc: read error!\n");
-        tmp = 0x00;
+#ifndef __MINGW32__		
+		tmp = (char)NULL;
+#else
+		tmp = 0x00;
+#endif
```

Comment:
This change shouldn't be needed. I'm pretty sure this is undoing something that was done in one of your earlier PRs.

Reply:
Fixed.